File: src/Entity/Reponse.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\ReponseRepository;
/**
 * @ORM\Entity(repositoryClass=ReponseRepository::class)
 */
class Reponse
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private int $id;

    /**
     * @ORM\Column(type="string", length=40)
     */
    private string $objectif;

    /**
     * @ORM\Column(type="string", length=40)
     */
    private string $texte;

    /**
     * @ORM\Column(type="integer", nullable=false)
     */
    private int $idrec;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getObjectif(): ?string
    {
        return $this->objectif;
    }

    public function setObjectif(string $objectif): self
    {
        $this->objectif = $objectif;

        return $this;
    }

    public function getTexte(): ?string
    {
        return $this->texte;
    }

    public function setTexte(string $texte): self
    {
        $this->texte = $texte;

        return $this;
    }

    public function getIdrec(): ?int
    {
        return $this->idrec;
    }

    public function setIdrec(int $idrec): self
    {
        $this->idrec = $idrec;

        return $this;
    }
}

File: src/com/reclamation/entities/Reponse.java
package com.reclamation.entities;

import java.util.Date;
import com.reclamation.utils.*;

public class Reponse {
    
    private int id;
     private int reclamation;
     private String objectif;
     private String texte;
     private int user;
    
    public Reponse() {}

    public Reponse(int id, int reclamation, String objectif, String texte, int user) {
        this.id = id;
        this.reclamation = reclamation;
        this.objectif = objectif;
        this.texte = texte;
        this.user = user;
    }

    public Reponse(int reclamation, String objectif, String texte, int user) {
        this.reclamation = reclamation;
        this.objectif = objectif;
        this.texte = texte;
        this.user = user;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
    
    public int getReclamation() {
        return reclamation;
    }

    public void setReclamation(int reclamation) {
        this.reclamation = reclamation;
    }
    
    public String getObjectif() {
        return objectif;
    }

    public void setObjectif(String objectif) {
        this.objectif = objectif;
    }
    
    public String getTexte() {
        return texte;
    }

    public void setTexte(String texte) {
        this.texte = texte;
    }
    
    public int getUser() {
        return user;
    }

    public void setUser(int user) {
        this.user = user;
    }
    
    
    
}